contracts/crowdsale/Crowdsale.sol

```diff
@@ -54,9 +54,10 @@ contract Crowdsale is ReentrancyGuard {
      * @param token Address of the token being sold
      */
     constructor (uint256 rate, address payable wallet, IERC20 token) public {
-        require(rate > 0);
-        require(wallet != address(0));
-        require(address(token) != address(0));
+        require(rate > 0, "Crowdsale: rate is 0.");
+        require(wallet != address(0), "Crowdsale: wallet address can only be non-zero address.");
```

Comment:
We should attempt to describe what went wrong, as opposed to the requirement, e.g. 'wallet address is zero'.